app/utils/themeUtils.js

```diff
@@ -1,6 +1,9 @@
 export const applySelectedTheme = (theme) => {
   if (__CLIENT__) {
-    document.documentElement.setAttribute('data-theme', theme);
+    document.documentElement.setAttribute(
```

Comment:
This will probably not work that well with the server side renderer :sweat_smile:

Reply:
Shouldn't this only run on the client due to the if above?

Reply:
Ahh, yeah, the placement of this comment is kinda awkward, I agree... :sweat_smile: 

SSR theme is set here: https://github.com/webkom/lego-webapp/blob/b78907fa7c575053d6d996588c3cb4cf0e9f3557/server/pageRenderer.js#L67, and to get auto working there will probably require some work. I have not tested this PR, so unsure what happen if that value is set to "auto", other than the fact that auto will not work properly. (eg. will it be all funky until light/dark is set, or will it look like "light" before the correct theme is applied).

Reply:
![Screenshot from 2021-05-24 10-40-26](https://user-images.githubusercontent.com/1467188/119321728-40db9b80-bc7d-11eb-9179-14d27e07288d.png)

Yeah, it will start loading like this (without abakus-logo), and then switch to correct light or dark theme on first render. It works, but feels a bit flashy.

Reply:
Replacing 'auto' with 'light' in SSR will help mitigate most issues, and only have the flash for users with auto while in dark mode. Then we can fix SSR later.

Reply:
I was also thinking that it would be great if the regular light and dark mode (not auto) still had the benefit of SSR. I will look into it